### DISCUSSION TITLE
What is the equivalent of `.. foo` in markdown?

### DISCUSSION THREAD

**Discussion by nikomatsakis** [executablebooks/MyST-Parser#989] [Q&A]
It would be really helpful to have a "cheat sheet" for comparing rst to markdown-- I didn't find one in the docs in an obvious place. I've got rst like this:

```
.. _fls_83182bfa9uqb:

Macros
======
```

and I can't tell what the equivalent is in markdown (if there is one). It is not, from what I can tell, a "directive" but rather a "role", but not an inline one... help!

**Comment by agoose77** [comment]
That is a directive, so the MyST equivalent would be
````
```{_fls_83182bfa9uqb}
```
````

**Reply by chrisjsewell** [reply]
No it's not 😅 it's a target

**Reply by agoose77** [reply]
@chrisjsewell :rofl: maybe I'll rethink answering discussions on the move[^nah]

[^nah]: Na, it'll happen again 

I'm going to delete this because it's just going to confuse things!

**Comment by agoose77** [comment]
@nikomatsakis now that I'm back on a full screen, as @chrisjsewell correctly identifies -- that is a _target_.

In MyST Markdown, we have a couple of ways of generating a target. The most frequently used variants are 
```markdown
(target)=

## A heading
```

and the `name` / `label` option in a directive:
````markdown
```{image} ...
:name: target
```
````
